# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux24/arch-ppc64-ioctl32.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Naur linux-2.4.30/arch/ppc64/kernel/ioctl32.c linux-2.4.30-new/arch/ppc64/kernel/ioctl32.c
--- linux-2.4.30/arch/ppc64/kernel/ioctl32.c 2005-01-19 09:09:36.000000000 -0500
+++ linux-2.4.30-new/arch/ppc64/kernel/ioctl32.c 2005-05-02 11:57:22.000000000 -0400
@@ -49,6 +49,7 @@
 #include <linux/cdrom.h>
 #include <linux/loop.h>
 #include <linux/auto_fs.h>
+#include <linux/auto_fs4.h>
 #include <linux/devfs_fs.h>
 #include <linux/tty.h>
 #include <linux/vt_kern.h>
